File: Python Exams/Basic Exam Collection/Basics Online Exam - 9 and 10 March 2019/02. Skeleton/02. Skeleton.py
control_min, control_sec, m_long ,sec_per_meters = [float(input()) for _ in range(4)]

control_time = control_min * 60 + control_sec
time_decrease = m_long / 120
total_decrease_time = time_decrease * 2.5
player_time = m_long / 100 * sec_per_meters - total_decrease_time

if player_time <= control_time:
    print(f'Marin Bangiev won an Olympic quota!\nHis time is {player_time:.3f}.')
else:
    print(f'No, Marin failed! He was {player_time - control_time:.3f} second slower.')





'''
1
20
1546
12
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 9 and 10 March 2019/02. Skeleton/02. Skeleton.js
function skeleton(...arg) {
    let [controlMin, controlSec, mLong, secPerMeters] = [...arg.map(Number)]

    let controlTime = controlMin * 60 + controlSec
    let timeDecrease = mLong / 120
    let totalDecreaseTime = timeDecrease * 2.5
    let playerTime = mLong / 100 * secPerMeters - totalDecreaseTime

    if (playerTime <= controlTime)
        return `Marin Bangiev won an Olympic quota!\nHis time is ${playerTime.toFixed(3)}.`
    return `No, Marin failed! He was ${(playerTime - controlTime).toFixed(3)} second slower.`
}

//console.log(skeleton(2, 12, 1200, 10))